tools/hierarchical/cluster.c

```
@@ -337,7 +337,7 @@ int main (int argc, char *argv [])
 				k++;
 			if (k)
 			    update_cp (binary ? (k / (float) n_total) : (k * d / (float) n_total), p1, p2);
-		    } else 
+		    } else
```

Clang (14.0.6)

[note] +1, nesting level increased to 5

(clang)